app/src/common/utils/referenceDictionary.js

```diff
@@ -29,4 +29,17 @@ export const referenceDictionary = {
   rpEpamNotice: 'https://privacy.epam.com/core/interaction/showpolicy?type=CommonPrivacyNotice',
   rpEpamCommonPolicy:
     'https://privacy.epam.com/core/interaction/showpolicy?type=CommonPrivacyPolicy',
+  indexSettingsDocs:
+    'https://reportportal.io/docs/analysis/AutoAnalysisOfLaunches#removegenerate-elasticsearch-index',
+  autoAnalysisDocs: 'https://reportportal.io/docs/analysis/AutoAnalysisOfLaunches',
+  similarItemsDocs: 'https://reportportal.io/docs/analysis/SearchForTheSimilarToInvestigateItems',
+  uniqueErrorsDocs: 'https://reportportal.io/docs/analysis/UniqueErrorAnalysis',
+  patternAnalysisDocs: 'https://reportportal.io/docs/analysis/PatternAnalysis',
+  emptyStatePatternAnalysisDocs: 'https://reportportal.io/docs/analysis/PatternAnalysis',
+  notificationsDocs:
+    'https://reportportal.io/docs/reportportal-configuration/ProjectConfiguration#e-mail-notifications',
+  emptyStateNotificationsDocs:
+    'https://reportportal.io/docs/reportportal-configuration/ProjectConfiguration#e-mail-notifications',
+  emptyStateIntegrationsDocs:
+    'https://reportportal.io/docs/reportportal-configuration/IntegrationViaPlugin',
```

Comment:
May create a separate object `docsReferences` to separate it from the common rarely changed links?